app/admin.js

```diff
@@ -114,12 +147,74 @@ const handleDeriveKey = function(args) {
   }
 };
 
-const run = co(function *() {
-  const args = parser.parseArgs();
+const handleVerificationGet = co(function *(args) {
+  const xpub = args.xpub;
+
+  const key = yield WalletKey.findOne({ xpub }).lean();
+
+  if (key === null) {
+    throw new Error(`Unable to find wallet key: ${xpub}`);
+  }
+
+  if (_.isUndefined(key.verificationInfo)) {
+    key.verificationInfo = '<N/A>'
+  }
+
+  // if there are multiple lines, this aligns each line under the first line
+  const formattedVerificationInfo = key.verificationInfo.replace(/\n/g, '\n\t\t\t');
+
+  console.log();
+  console.log(`Key:\t\t\t${key.xpub}`);
+  console.log(`Master Key:\t\t${key.masterKey}`);
+  console.log(`User Email:\t\t${key.userEmail}`);
+  console.log(`Verification Info:\t${formattedVerificationInfo}`);
+  console.log('Custom data:');
+  console.log(JSON.stringify(key.custom, null, 2));
```

Comment:
Is this okay if custom doesnt exist?

Reply:
It will always exist as at least an empty object - but it could be a little cleaner indeed. Fixed.